pyproject.toml

```diff
@@ -1,13 +1,21 @@
-[tool.poetry]
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+packages = ["pytest_spec"]
+
+[project]
 name = "pytest-spec"
 version = "4.0.0"
+authors = [
+    {name = "Pawel Chomicki", email = "pawel.chomicki@gmail.com"},
+]
 description = "Library pytest-spec is a pytest plugin to display test execution output like a SPECIFICATION."
 readme = "README.md"
-authors = ["Pawel Chomicki <pawel.chomicki@gmail.com>"]
-license = "GPL-2.0-or-later"
-homepage = "https://github.com/pchomik/pytest-spec"
-repository = "https://github.com/pchomik/pytest-spec"
+requires-python = ">=3.7"
 keywords = ["pytest", "test", "unittest", "spec"]
+license = {text = "GPL-2.0-or-later"}
 classifiers = [
```

Comment:
We can remove `"Programming Language :: Python :: 3.7"` from here, and add `"Programming Language :: Python :: 3.12"`.